sdk/core/core-auth/rollup.base.config.js

```diff
@@ -43,8 +43,13 @@ export function nodeConfig(test = false) {
 
     // mark assert as external
     baseConfig.external.push("assert");
+
+    // Disable tree-shaking of test code.  In rollup-plugin-node-resolve@5.0.0, rollup started respecting
```

Comment:
Noticed that `core-auth` didn't have the tree-shaking test fix that was introduced in https://github.com/Azure/azure-sdk-for-js/pull/3997 so I added it here so that tests will run correctly.